packages/Debian/DEBIAN/postinst

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+##--------------------------------------------------------------------
+## Copyright (c) 2018 Dianomic Systems
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+
+##--------------------------------------------------------------------
+##
+## @postinst DEBIAN/postinst
+## This script is used to execute post installation tasks.
+##
+## Author: Amarendra K Sinha
+##
+##--------------------------------------------------------------------
+
+set -e
+
+set_files_ownership () {
+    chown -R root:root /usr/local/foglamp/python/foglamp/plugins/south/systeminfo
+}
+
+add_service () {
```

Comment:
We should not add the service in post install.

- We do not know what the user wants to create
- FogLAMP may not be running
- The API might have authentication turned on, so this would not work

Reply:
@MarkRiddoch though I have removed the lines from postinst, but please note that,
- Not clear by "We do not know what the user wants to create". Do you mean that user may want to have a different service name etc?
- Preinst already checks if Foglamp is running, so shall I remove this check also?
- The prevalent Debian package behaviour expects the package to be up and running when installed. Removing add_service() call contradicts this.